docs/source/python/api/tables.rst

```diff
@@ -59,7 +59,92 @@ Dataframe Interchange Protocol
 Tensors
 -------
 
+PyArrow supports both dense and sparse tensors. Dense tensors store all data values explicitly, while sparse tensors represent only the non-zero elements and their locations, making them efficient for storage and computation.
+
+Dense Tensors
+-------------
```

Comment:
Could you use `^^^...`?
See also: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections